content/docs/command-reference/plots/index.md

```diff
@@ -1,7 +1,7 @@
 # plots
 
-A set of commands to visualize and compare _plot metrics_ in structured files
-(JSON, YAML, CSV, or TSV): [show](/doc/command-reference/plots/show),
+A set of commands to visualize and compare _plot metrics_:
+[show](/doc/command-reference/plots/show),
 [diff](/doc/command-reference/plots/diff), and
 [modify](/doc/command-reference/plots/modify).
```

Comment:
I think we should clarify here (in the usage block below) and in https://dvc.org/doc/command-reference/plots/modify that it only works with structured file plots.

Reply:
Related https://github.com/iterative/dvc/issues/6669

Reply:
@jorgeorpinel added context to `modify`
Also created:
 https://github.com/iterative/dvc/pull/6742